### DISCUSSION TITLE
Error occurs in installation of FEPX

### DISCUSSION THREAD

**Discussion by xchengood** [neperfepx/FEPX#49] [Q&A] [answered]
Hi FEPX experts or users,

I got the following error when installing FEPX on Mac. Could anyone help me solve it? Thank you.
![image](https://user-images.githubusercontent.com/74226455/143094305-0fa524d3-1428-4e76-917c-a66a87fe6dec.png)

**Comment by mkasemer** [comment, marked as answer]
What version of OpenMPI are you using?

**Comment by xchengood** [comment]
> What version of OpenMPI are you using?

Hi, how to check the version OpenMPI? Thanks.

**Reply by mkasemer** [reply]
`mpirun --version` is one way.

**Reply by mkasemer** [reply]
@xchengood, please reply to comments rather than creating new comments, as done below.

The error you posted above is indicating that it cannot find `mpif-config.h`, a file found with OpenMPI that is necessary for compilation of the FEPX's parallel processing library. This either indicates that you do not have OpenMPI installed, or you do not have your paths set properly. Unfortunately, I am unfamiliar with HYDRA, and cannot help you regarding this specific system configuration. However, as a note, OpenMPI can be installed via Brew.

**Comment by xchengood** [comment]
> `mpirun --version` is one way.

Thank you, I got the following message.
<img width="749" alt="Screen Shot 2021-11-23 at 3 19 27 PM" src="https://user-images.githubusercontent.com/74226455/143130896-a8c84d88-dc76-4b9d-a3f8-5fc154e5c793.png">

**Comment by xchengood** [comment]
> @xchengood, please reply to comments rather than creating new comments, as done below.
> 
> The error you posted above is indicating that it cannot find `mpif-config.h`, a file found with OpenMPI that is necessary for compilation of the FEPX's parallel processing library. This either indicates that you do not have OpenMPI installed, or you do not have your paths set properly. Unfortunately, I am unfamiliar with HYDRA, and cannot help you regarding this specific system configuration. However, as a note, OpenMPI can be installed via Brew.

Appreciate your information, letting me clearly what the error is. Let me try to solve it.
